stackdriver/client_test.go

```diff
@@ -114,3 +117,43 @@ func TestEmptyRequest(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func TestResolver(t *testing.T) {
+  grpcServer := grpc.NewServer()
```

Comment:
The indentation is off, please run `go fmt`. I suggest to run `make` from time to time, especially before pushing a revision to review, because it will run additional janitorial actions on your code. Namely, it'll format the code, and run the static code checker.